packages/features/form-builder/FormBuilderField.tsx

```diff
@@ -333,8 +334,8 @@ export const ComponentForField = ({
   }
 
   if (componentConfig.propsType === "variants") {
-    const variantsConfig = getVariantsConfig(field);
-    if (!variantsConfig) {
+    const translatedVariantsConfig = getTranslatedVariantsConfig(field, t);
```

Comment:
Passing the translatedVariantsConfig directly so that Components don't need to do the translation.